authentication/auth.go

```diff
@@ -375,9 +390,11 @@ func GenerateTokenFromPost(postValues url.Values) (string, string) {
 		claims[key] = v
 	}
 
-	requiredMetadata, error := GetRequiredMetadata(launcherSchema)
-	if error != "" {
-		return "", fmt.Sprintf("GetRequiredMetadata failed err: %v", error)
+	log.Println("hello", schemaClaims, "goodbye")
```

Comment:
should remove this debug comment

Reply:
Oops! Forgot about that one, nice catch. Cheers.